frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -100,6 +135,23 @@ export const createBuildConfig = (
       break;
   }
 
+  let webhookTriggerData;
+  if (gitType === GitTypes.unsure) {
+    webhookTriggerData = {
+      type: 'Generic',
```

Comment:
in 3.11 the Generic webhook is always added: https://github.com/openshift/origin-web-console/blob/4534d447667d928ed08b8eed3ce7b1a450d9a2d8/app/scripts/services/applicationGenerator.js#L288-L295